src/libs/Clipboard/index.native.ts

```diff
@@ -0,0 +1,19 @@
+import RNCClipboard from '@react-native-community/clipboard';
+import {SetString, Clipboard} from './types';
+
+/**
+ * Sets a string on the Clipboard object via @react-native-community/clipboard
+ */
+const setString: SetString = (text) => {
+    RNCClipboard.setString(text);
+};
+
+const clipboard: Clipboard = {
+    setString,
+
+    // We don't want to set HTML on native platforms so noop them.
+    canSetHtml: () => false,
+    setHtml: () => {},
```

Comment:
What do you guys think about adding `noop` function to the CONST file as we've done with `EMPTY_OBJECT`, so it is defined once instead of creating function every time?

Reply:
It's understandable if that is out of the scope of this PR 👍

Reply:
Looks like a good idea :), not for this PR though